plasmapy/utils/decorators/tests/test_validators.py

```diff
@@ -554,3 +542,9 @@ def test_decorator_func_def(self, mock_vq_class):
                 # reset
                 mock_vq_class.reset_mock()
                 mock_foo.reset_mock()
+
+
+def test_no_ImplicitUnitConversionWarning_on_eV_input():
+    with pytest.warns(None) as record:
+        Debye_number(1 * u.eV, 1e20 * u.m ** -3)
+    assert not record, f"Warnings raised: {[warning.message for warning in record]}"
```

Comment:
Is this a placeholder to see if your changes got ride of the `ImplicitUnitConversionWarning`?  In the end I don't think we need to explicitly test this.

Reply:
I started out with this one; I think it's fine to keep this here as a regression test.